### ISSUE TITLE
ValueError: Invalid boundary in multipart form: b''

### ISSUE THREAD

**Issue by yohanboniface** [yohanboniface/falcon-multipart#8]
```
[2017-05-11 15:27:24 +0200] [30466] [ERROR] Error handling request /search/csv
Traceback (most recent call last):
  File "/srv/addok/venv/lib/python3.4/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/srv/addok/venv/lib/python3.4/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/srv/addok/venv/lib/python3.4/site-packages/falcon/api.py", line 180, in __call__
    process_request(req, resp)
  File "/srv/addok/venv/lib/python3.4/site-packages/falcon_multipart/middleware.py", line 20, in process_request
    form = self.parse(stream=req.stream, environ=req.env)
  File "/srv/addok/venv/lib/python3.4/site-packages/falcon_multipart/middleware.py", line 10, in parse
    return self.parser(fp=stream, environ=environ)
  File "/usr/lib/python3.4/cgi.py", line 559, in __init__
    self.read_multi(environ, keep_blank_values, strict_parsing)
  File "/usr/lib/python3.4/cgi.py", line 681, in read_multi
    raise ValueError('Invalid boundary in multipart form: %r' % (ib,))
ValueError: Invalid boundary in multipart form: b''
```

**Comment by alexalv** [comment]
Encountered the same error, the problem in my case was in `Content-Type` header which I was setting explicitly on the client side. [Related StackOverflow question](https://stackoverflow.com/questions/18590630/xmlhttprequest-multipart-form-data-invalid-boundary-in-multipart) 
**tldr**: Do not set `Content-Type` header on the client side.
